src/main/java/com/example/SuperHero.java

```diff
@@ -0,0 +1,18 @@
+package com.example;
+
+public class SuperHero {
+
+    private final String name;
+
+    public SuperHero(String name) {
+        this.name = name;
+    }
+
+    public void saveTheWorld() {
+        System.out.println(this.name + " is saving the world.");
```

Comment:
We're never using these default implementations, so we should either start using them or make this class abstract/an interface